buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ForbiddenApisCliTask.java

```diff
@@ -51,7 +51,8 @@
     private JavaVersion targetCompatibility;
     private FileCollection classesDirs;
     private SourceSet sourceSet;
-    private String javaHome;
+    // This needs to be an object so it can hold Groovy GStrings
+    private Object javaHome;
```

Comment:
Is there a reason for not using `groovy.lang.GString` here as a specific type?

Reply:
Forget the question, I assume that class is still used in places where there is passed in a string directly.